cmd/testdata/status-local-version-stage

```diff
@@ -1,11 +1,11 @@
 Package: foo
 Owner: team
 Package Versions:
-+-------------+--------------------------------+-------------------+-------+-----------------+
-| ENVIRONMENT |            VERSION             |      RELEASE      | TITLE |   DESCRIPTION   |
-+-------------+--------------------------------+-------------------+-------+-----------------+
-| Local       | 2.0.0-rc1                      | Release Candidate | Foo   | Foo integration |
-+-------------+--------------------------------+-------------------+-------+-----------------+
-| Production  | 1.1.0-beta1 (1.0.0, 1.0.1,     | Beta              | Foo   | Foo integration |
-|             | 1.0.2)                         |                   |       |                 |
-+-------------+--------------------------------+-------------------+-------+-----------------+
++-------------+----------------------------+-------------------+-------+-----------------+
+| ENVIRONMENT |          VERSION           |      RELEASE      | TITLE |   DESCRIPTION   |
++-------------+----------------------------+-------------------+-------+-----------------+
+| Local       | 2.0.0-rc1                  | Release Candidate | Foo   | Foo integration |
++-------------+----------------------------+-------------------+-------+-----------------+
+| Production  | 1.1.0-beta1 (1.0.0, 1.0.1, | Beta              | Foo   | Foo integration |
+|             | 1.0.2)                     |                   |       |                 |
++-------------+----------------------------+-------------------+-------+-----------------+
```

Comment:
Wrapping seems to change in 1.0, I haven't managed to mimic old behaviour, but the new looks better.